tests-js/tutor/users.json

```diff
@@ -6,4 +6,9 @@
   "username": "content",
   "password": "password",
   "tags": ["content-analyst"]  
-}]
\ No newline at end of file
+}, {
+  "username": "student01",
+  "password": "password",
+  "tags": ["student"]
+	}
+]
```

Comment:
The usernames and passwords need to be removed from the repos. These values should be stored in system (environment) variables or an environment file; there is a template file at `/test-automation/tests-js/.env.example` - make a copy of the file, rename the copy `.env` and put the values in there.